package.json

```diff
@@ -27,7 +27,8 @@
     "react-native",
     "expo",
     "@jellyfish-dev/react-native-client-sdk",
-    "MembraneWebRTC"
+    "MembraneWebRTC",
+    "Fishjam"
   ],
   "repository": "https://github.com/membraneframework/react-native-client-sdk",
```

Comment:
Similarly, could you please update the links to https://github.com/fishjam-dev/react-native-client-sdk?

Reply:
Also, the package name is still in `jellyfish` namespace, as well as the keywords point to it (line 29). If this is intended to be fixed in another PR you can ignore it.

Reply:
Yes, all Jellyfish references will updated in separate PR (it requires a little more work)